src/objects/zcl_abapgit_object_devc.clas.abap

```diff
@@ -419,18 +419,32 @@ CLASS zcl_abapgit_object_devc IMPLEMENTATION.
         zcx_abapgit_exception=>raise_t100( ).
       ENDIF.
 
-      li_package->save(
-        EXPORTING
-          i_suppress_dialog     = abap_true
-        EXCEPTIONS
-          object_invalid        = 1
-          object_not_changeable = 2
-          cancelled_in_corr     = 3
-          permission_failure    = 4
-          unexpected_error      = 5
-          intern_err            = 6
-          OTHERS                = 7 ).
+      TRY.
+          CALL METHOD li_package->('SAVE')
+            EXPORTING
+              i_suppress_dialog     = abap_true
+            EXCEPTIONS
+              object_invalid        = 1
+              object_not_changeable = 2
+              cancelled_in_corr     = 3
+              permission_failure    = 4
+              unexpected_error      = 5
+              intern_err            = 6
+              OTHERS                = 7.
 
+        CATCH cx_root.
```

Comment:
let's use `CATCH cx_sy_dyn_call_param_not_found.`

Reply:
if you don't mind, could you change the other catch cx_root in this class, too? thanks

Reply:
@mbtools Yes, i did but checks have failed.

Reply:
nvm my bad, i put an extra single quote by mistake. should be fine now.